ale_linters/terraform/checkov.vim

```diff
@@ -0,0 +1,52 @@
+" Author: Thyme-87 <thyme-87@posteo.me>
+" Description: use checkov for providing warnings via ale
+
+call ale#Set('terraform_checkov_executable', 'checkov')
+call ale#Set('terraform_checkov_options', '-o json --quiet')
```

Comment:
Handler depends on the output being json. If a user defines this option in his/her config and do not specify these options it will break the linter.  The "-o json --quiet" options should be added to the command in the GetCommand() method always.